init.js

```diff
@@ -0,0 +1,27 @@
+let tetris = require('./tetris');
+module.exports = {
+    init: function () {
+        let fieldSIze = { x: 5, y: 5 };
+        let newTetris = new tetris.Tetris(fieldSIze, this.stopTick());
```

Comment:
you are passing stopTick function incorrectly

Reply:
I don't know, how I must transfer function stopTick :(
I went through a lot of options.
Via module.exports.init.stopTick
Via module.exports.init.stopTick()
Via init.stopTick()
Via init.stopTick
Via stopTick
Via stopTick()
All varianits is wrong:(